api_tests/nodes/views/test_node_list.py

```diff
@@ -346,6 +376,31 @@ def test_incorrect_filtering_field_not_logged_in(self):
         assert_equal(len(errors), 1)
         assert_equal(errors[0]['detail'], "'notafield' is not a valid field for this endpoint.")
 
+    def test_filtering_on_root(self):
+        # Build up a family of nodes
+        node_structure = [5, 2, [1, 2], 3, [10, 2], 2, [4, [2, 2], 6]]
+        render_generations_from_node_structure_list(self.project_one, node_structure)
+        # create some unrelated projects
+        ProjectFactory(title="Road Dogg Jesse James", is_public=True)
+        ProjectFactory(title="Badd *** Billy Gunn", is_public=True)
+
+        url = '/{}nodes/?filter[root]={}'.format(API_BASE, self.project_one._id)
+
+        res = self.app.get(url, auth=self.user_one.auth)
+        assert_equal(res.status_code, 200)
+        assert_equal(res.json['links']['meta']['total'], 42)
+
+    def test_filtering_on_null_parent(self):
+        # Build up a family of nodes not to be included
+        node_structure = [2, [1, 2]]
+        render_generations_from_node_structure_list(self.project_one, node_structure)
+
+        url = '/{}nodes/?filter[parent]=null'.format(API_BASE)
+
+        res = self.app.get(url, auth=self.user_one.auth)
+        assert_equal(res.status_code, 200)
+        assert_equal(res.json['links']['meta']['total'], 4)
```

Comment:
If I understand this test correctly, `filter[parent]=null` results in no filtering at all. Is this the desired behavior, @brianjgeiger @erinspace ? 

As a user, I think I'd expect `filter[parent]=null` to mean "filter on nodes with no parents", i.e. root nodes only.